MetaMask/metamask-extension
# UX: Show native currency warning when unconfirmed symbol

## **Description**

This adds a warning label in the currency symbol position when a native token has an incorrect symbol

## **Related issues**

Fixes: https://app.zenhub.com/workspaces/metamask-extension-63529dce65cbb100265a3842/issues/gh/metamask/metamask-planning/1652

## **Manual testing steps**

1. Add a network with chain `137` (Polygon, but call it "Polyfraud") with incorrect symbol (`ETH`)
2. Switch to Polyfraud network
3. See red "danger" icon 
4. Hover over said icon
5. See the modal with important information about potential fraud

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

<img width="716" alt="SCR-20231121-nbrz" src="https://github.com/MetaMask/metamask-extension/assets/46655/be1bccc3-5788-4233-a2fb-09fbf77713a3">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
